dracut/anaconda-diskroot

```diff
@@ -64,21 +64,26 @@ fi
 # retrigger this script, and we'll mount the disk as normal.
 if str_starts "$kickstart" "cdrom" && [ ! -e /tmp/ks.cfg.done ]; then
     if dev_is_cdrom "$dev"; then
-        > /tmp/anaconda-on-cdrom
+        # create empty marker file
+        true > /tmp/anaconda-on-cdrom
```

Comment:
That came to my mind as well - I'm guessing it might not be available in the initrd or have behaviour different from this somehow ?

In any case, it really is not a big readability improvement even with this "fix" and as it's a lot of changes anyway I'll try to check if touch is available in the initrd environment and do use it instead if possible.

Reply:
The `touch` tool can be easily installed in `module-setup.sh`. Another alternative is to use `echo "" > /file/path`.

Reply:
Please, do not install `touch` into initrd unless it's really necessary (to avoid growing the size of images).
Both `true >` and `echo "" >` are IMO fine with the Martin's comment.

Reply:
What about `: > file`?